src/Microsoft.Diagnostics.Tools.RuntimeClient/Microsoft.Diagnostics.Tools.RuntimeClient.csproj

```diff
@@ -13,13 +11,4 @@
     <PackageTags>Diagnostic</PackageTags>
     <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
   </PropertyGroup>
-
-  <PropertyGroup>
```

Comment:
Why was this removed to move forward the major version?

Reply:
Cleanup in the case of trace event (packaging double writes), and the other property was repeated.